internal/extproc/mocks_test.go

```diff
@@ -168,3 +170,60 @@ func (m mockExternalProcessingStream) SendMsg(any) error { panic("TODO") }
 func (m mockExternalProcessingStream) RecvMsg(any) error { panic("TODO") }
 
 var _ extprocv3.ExternalProcessor_ProcessServer = &mockExternalProcessingStream{}
+
+type mockChatCompletionMetrics struct {
```

Comment:
let's add comments like elsewhere
```suggestion
// mockChatCompletionMetrics implements ...
type mockChatCompletionMetrics struct {
```

Reply:
Done